src/App/App.js

```diff
@@ -13,7 +12,6 @@ class App extends Component{
     return (
      <>
       <Homepage />
-      <WelcomingSection />
```

Comment:
by removing the welcomingSection and inserting it into the home page it allows us to have more control when we get to routing